src/main/java/edu/njit/jerse/ashe/utils/JavaCodeCorrector.java

```diff
@@ -19,6 +8,19 @@
 import java.util.concurrent.TimeoutException;
 import java.util.regex.Pattern;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
Rather than making these kind of changes by hand, it would be better to integrate an autoformatter into the build system that will do this automatically. My preference for java is https://github.com/diffplug/spotless, but any auto-formatter will do. If you decide to do this, make a separate PR that contains only the build script code to run the formatter and the changes that the formatter makes (i.e., no semantic changes to the code), so that the formatting changes don't have to be reviewed closely.

Reply:
The imports were being managed by the IDE, not sure why it changed the order